internal/ui/app.go

```
@@ -337,41 +399,56 @@ func checkForUpdate(currentVersion string) tea.Cmd {
 	}
+	return cmds
+}
 
-	sort.Slice(all, func(i, j int) bool {
-		return all[i].Name < all[j].Name
-	})
-
-	manager.SaveScanCache(all)
-	return scanDoneMsg{pkgs: all}
+// startFreshScan emits the start message followed by per-manager scan commands.
+func startFreshScan() tea.Cmd {
```

golangci-lint (2.11.4)

[error] func startFreshScan is unused

(unused)